src/main/java/org/springframework/integration/splunk/event/SplunkEvent.java

```diff
@@ -431,10 +247,17 @@ private void addThrowableObject(Throwable throwable, int stackTraceDepth) {
 	 */
 	public void addPair(String key, String value) {
 		Assert.notNull(key, "key cannot be null");
-		if (quoteValues)
-			this.eventMessage.append(key).append(KVDELIM).append(QUOTE).append(value).append(QUOTE).append(PAIRDELIM);
-		else
-			this.eventMessage.append(key).append(KVDELIM).append(value).append(PAIRDELIM);
+		
+		//start json
+		if (this.eventMessage.length() == 0){
```

Comment:
Whitespace before `{`.
The code style is here: https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style
But yes, it is minor and I definitely can fix that on merge.